com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviourUpdater.cs

```diff
@@ -6,28 +6,13 @@ namespace MLAPI
     public class NetworkBehaviourUpdater
     {
         private HashSet<NetworkObject> m_Touched = new HashSet<NetworkObject>();
-
-        /// <summary>
-        /// Stores the network tick at the NetworkBehaviourUpdate time
-        /// This allows sending NetworkVariables not more often than once per network tick, regardless of the update rate
-        /// </summary>
-        public ushort CurrentTick { get; set; }
-
+        
 #if DEVELOPMENT_BUILD || UNITY_EDITOR
         private ProfilerMarker m_NetworkBehaviourUpdate = new ProfilerMarker($"{nameof(NetworkBehaviour)}.{nameof(NetworkBehaviourUpdate)}");
 #endif
 
         internal void NetworkBehaviourUpdate(NetworkManager networkManager)
         {
-            // Do not execute NetworkBehaviourUpdate more than once per network tick
-            ushort tick = networkManager.NetworkTickSystem.GetTick();
```

Comment:
@SamuelBellomo I removed this from NetworkBehaviour and now removing this from here as well. This will be called during the network tick now, (In theory multiple times per frame but only once per network tick). This is because each a network variable could be updated by user code and we don't want to lose that information (in most cases irrelevant unless the frame rate < tick rate). Hope that makes sense.

Reply:
I'm fine with it, but you might want to git blame deeper on this, I'm not the original writer of that code, I copy pasted it when I moved this from NetworkBehaviour.

Reply:
👋  
I think I'm the original writer of that code.
The aim is really as commented. It needs to early-out so that the rest of the update happens only once per tick.

Reply:
yep, all good. Was just providing data on the "I'm not the original writer of that code" part.